packages/@sanity/types/src/portableText/types.ts

```diff
@@ -1,8 +1,8 @@
-export interface Block<O = Span> {
+export interface Block<TChild = Span> {
   _type: 'block'
```

Comment:
This `_type: 'block'` should be string as well as we now supports custom block type names.

Reply:
I can confirm that it's not validating properly on hoisted block types.

Reply:
I changed it.